crates/sdk/src/lib.rs

```diff
@@ -0,0 +1,20 @@
+use async_trait::async_trait;
+pub use configuration::{NetworkConfig, NetworkConfigBuilder, RegistrationStrategy};
+pub use orchestrator::{errors::OrchestratorError, network::Network, AddNodeOpts, Orchestrator};
+use provider::NativeProvider;
+use support::{fs::local::LocalFileSystem, process::os::OsProcessManager};
+
+#[async_trait]
+pub trait Spawner {
```

Comment:
I think we can rethink this `naming`, something like `NetworkConfigSpawnExt` or just `NetworkConfigExt`. 
@wirednkod / @l0r1s / @pgherveou wdyt?

Reply:
Yes, we can do this temporarily, not a issue to change this in the future :+1:

Reply:
👍  NetworkConfigExt sounds a bit better